action/main.py

```diff
@@ -24,6 +24,8 @@
 if VERSION:
     if VERSION.startswith("@"):
         req[0] = f"git+https://github.com/akaihola/darker{VERSION}#egg={req[0]}"
+    elif VERSION.startswith(("~", "=", "<", ">")):
+        req[1] += VERSION
```

Comment:
Shouldn't this be `req[0] += VERSION`? This will cause a IndexError if I am understanding the code correctly.

Reply:
I think you are correct. @akaihola, we have missed this one.

Reply:
I remember abandoning the idea to attempt creating unit tests for the action implementation. Would have become handy after all... :smirk: